### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]:NVDA and Jaws are not announcing the content present in  select subscriptions window sequentially . 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1511]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to Tools menu, point to "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Interactive" radio button, and navigate to "Sign In" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. In the Select Subscriptions dialog , Press "Insert+B" to read  the entire window then observe the NVDA and Jaws Behavior

**Expected Result:**
NVDA and JAWS Should announce in the following  manner

- Select Subscriptions Dialog
- Table
   Subscription name header
   Subscription ID header
   check box  checked
   DevDiv Key Vault
   bd62906c-0a81-43c3-a2f8-126e4cf66ada
  check box  checked
  HDInsight VS Tool Team
  7f31cba8-b597-4129-b158-8f21a7395bd0
  check box  checked
  Java Dev Experience Tests with TTL = 7 Days
  ffa52f27-be12-4cad-b1ea-c2c241b6cceb
- Refresh  button  Refresh
- Select  button
- Cancel  button

**Actual Result:**
**NVDA Behavior:**
NVDA is announicng as
Select Subscriptions  dialog
table
check box  checked
DevDiv Key Vault
bd62906c-0a81-43c3-a2f8-126e4cf66ada
check box  checked
HDInsight VS Tool Team
7f31cba8-b597-4129-b158-8f21a7395bd0
check box  checked
Java Dev Experience Tests with TTL = 7 Days
ffa52f27-be12-4cad-b1ea-c2c241b6cceb
Subscription name
Subscription ID
Refresh  button  Refresh
filler
Select  button
Cancel  button
**Jaws Behavior:**
Select Subscriptions dialog
Subscription name Subscription ID 
Select Button
table
Not Selected true
Not Selected true
Not Selected Java Dev Experience Tests with TTL = 7 Days
Refresh Button Pressed
Select Button
Select Subscriptions dialog
Subscription name Subscription ID 
Select Button

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![select subscription](https://user-images.githubusercontent.com/22956130/36149690-b7b7d480-10e6-11e8-8506-f180ee011cdb.jpg)
![hierarchy is incorrect for select subscriptions dialog](https://user-images.githubusercontent.com/22956130/36149696-bf25146c-10e6-11e8-9443-98a05344bc58.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749984
